src/middlewares/irrelevant-entities-middleware.ts

```diff
@@ -0,0 +1,21 @@
+import { injectable } from 'inversify';
+import { isContainsIrrelevant } from '../common/query-irr-result';
+import { Middleware, RequestMiddlewareParams, ResponseMiddlewareParams } from './middleware';
+
+@injectable()
+export class IrrelevantEntitiesMiddleware implements Middleware {
+    preResolve({ args }: RequestMiddlewareParams) {
+        return;
+    }
+
+    postResolve(params: ResponseMiddlewareParams): any {
+        if (!params.result) {
+            return;
+        }
+        if (isContainsIrrelevant(params.result)) {
```

Comment:
could'nt you just check `params.result.irrelevantEntities`?

Reply:
I could, but what if someone called a field on their entity as "irrelevantEntities"? it will automatically move it to the extensions.
Anyways it similar to a typeof check, I can change it to if(params.result.irrelevantEntities) if you want